src/playground/components/Configuration.js

```diff
@@ -184,7 +184,7 @@ export default function Configuration({ rulesMeta, eslintVersion, onUpdate, opti
                         <Select
                             isClearable
                             isMulti
-                            defaultValue={ECMAFeaturesOptions.filter(ecmaFeatureName => options.parserOptions.ecmaFeatures[ecmaFeatureName.value])}
+                            defaultValue={ECMAFeaturesOptions.filter(ecmaFeatureName => options.parserOptions.ecmaFeatures?.[ecmaFeatureName.value])}
```

Comment:
Given that this isn't the only problem (see https://github.com/eslint/eslint.org/issues/450#issuecomment-1627164477), I think it would be better to leave this as is, and instead normalize received configs here:

https://github.com/eslint/eslint.org/blob/6903c95c2eb261e9c6340c9beaea09aec33e9de9/src/playground/App.js#L25-L27